app/filters.js

```diff
@@ -26,6 +27,21 @@ module.exports = (env) => {
     }
   }
 
+  /**
+   * GOV.UK style dates
+   * @type {String} str
+   */
+  filters.govukDate = date => {
+    return moment(date).format('D MMMM YYYY')
+  }
+
+  filters.govukDateAtTime = date => {
```

Comment:
how about `govukDateTime`?

Reply:
I don't think that's quite what this exposes - so suggest leaving for now.